legacyreleases/LICENSE.md

```diff
@@ -0,0 +1,27 @@
+<!-- SPDX-FileCopyrightText: Copyright 2023 Arm Limited and/or its affiliates <open-source-office@arm.com> -->
+<!-- SPDX-License-Identifier: CC-BY-SA-4.0 AND Apache-Patent-License -->
+
+# License
```

Comment:
I wonder if this does more harm than good. None of the documents are licensed under Creative Commons, and I don't think we need a Creative Commons license for the readme.

Reply:
In the legacyreleases folder I left only the README file, which is itself under Apache 2.0. So no mention of CC anymore under this folder.